.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  printWidth: 100,
+  parser: 'flow',
+  tabWidth: 2,
+  singleQuote: true,
+  bracketSpacing: true,
+  arrowParens: 'avoid',
```

Comment:
In my experience `always` is nicer to work with because it's consistent.
You never have to add parenthesis because you add an argument or use destructuring.

```javascript
// let's say you use `avoid` configuration and omit parenthesis like below
const fn = a => a

// later you need update the function because it uses a second argument, you must add the parenthesis.
// prettier certainly made you lazy so you'll forgot and get a syntaxError (true story)
const fn = (a, b) => a + b

// same thing happens when you want to use destructuring
const fn = ({name}) => name
```

My advice is to prefer `always`.

Reply:
I agree with you on this, but then why did you chose on the player to remove semicolons ?

```javascript
if (
  extension.smth && extension.vmap ||
  extension.getExpression()
) {
  extension.doSomething();
  extension.doAnotherThing();
}
```
can be read in the blink of an eye, but you'll have to do a double take on this:
```javascript
if (
  extension.smth && extension.vmap ||
  extension.getExpression()
) {
  extension.doSomething()
  extension.doAnotherThing()
}
```